### PR TITLE
Add EI documentation within README

### PR DESCRIPTION
**Pull request by ChoiByungWook**
Adding section to README in regards to Elastic Inference.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,12 +5,9 @@ SageMaker MXNet Container
...
 For information on running MXNet jobs on Amazon SageMaker, please refer to the `SageMaker Python SDK documentation <https://github.com/aws/sagemaker-python-sdk>`__.
 
-Table of Contents
+Table Of Contents
 -----------------
-
-#. `Getting Started <#getting-started>`__
-#. `Building your Image <#building-your-image>`__
-#. `Running the tests <#running-the-tests>`__
+.. contents:: \
```
Comment (laurenyu) [inline README.rst]: GitHub doesn't like rendering RST directives nicely sometimes, and I think this is one of those cases. My suggestion would be to instead do:

```rst
-----------------
Table of Contents
-----------------

.. contents::
  :local:
```

also, the "of" should not be capitalized

File: README.rst [hunk 2]
```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
...
                                       --tag 1.3.0-cpu-py3
 
+Amazon Elastic Inference with MXNet in SageMaker
+------------------------------------------------
+`Amazon Elastic Inference <https://aws.amazon.com/machine-learning/elastic-inference/>`__ allows you to to attach
+low-cost GPU-powered acceleration to Amazon EC2 and Amazon SageMaker instances to reduce the cost running deep
+learning inference by up to 75%. Currently, Amazon Elastic Inference supports TensorFlow, Apache MXNet, and ONNX
+models, with more frameworks coming soon.
+
+Support for using MXNet with Amazon Elastic Inference in SageMaker is supported in the public `SageMaker MXNet containers <https://github.com/aws/sagemaker-mxnet-container>`__.
```
Comment (laurenyu) [inline README.rst]: if we're already in this repo, do we need to link to it?

File: README.rst [hunk 3]
```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
...
+* For information on how Amazon Elastic Inference works, see `How EI Works <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-how-it-works>`__.
+* For more information in regards to using Amazon Elastic Inference in SageMaker, see `Amazon SageMaker Elastic Inference <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html>`__.
+* For notebook examples on how to use Amazon Elastic Inference with MXNet through the Python SDK in SageMaker, see `EI Sample Notebooks <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-intro-sample-nb>`__.
+
+SageMaker Elastic Inference MXNet container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Amazon Elastic Inference is designed to be used with AWS enhanced versions of TensorFlow serving or Apache MXNet. These enhanced
+versions of the frameworks are automatically built into containers when you use the Amazon SageMaker Python SDK, or you can
+download them as binary files and import them into your own Docker containers. The enhanced MXNet binary are available on Amazon S3, `here <https://s3.console.aws.amazon.com/s3/buckets/amazonei-apachemxnet/>`__.
```
Comment (laurenyu) [inline README.rst]: * s/binary/binaries (for the last one)
* I'd remove "here" and make the link for "available on Amazon S3"

File: README.rst [hunk 4]
```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
...
+
+The `SageMaker MXNet containers <https://github.com/aws/sagemaker-mxnet-container>`__ with Amazon Elastic Inference support were built utilizing the
```
Comment (laurenyu) [inline README.rst]: again, don't think you need to link to the repo here

File: README.rst [hunk 5]
```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
...
+same instructions listed `above <https://github.com/aws/sagemaker-mxnet-container#building-images>`__ with the
+`CPU Dockerfile <https://github.com/aws/sagemaker-mxnet-container/blob/master/docker/1.3.0/final/Dockerfile.cpu>`__ starting at MXNet version 1.3.0 and above.
+
+The only difference being that the enhanced version of MXNet was passed in for the ``framework_installable`` build-arg.
```
Comment (laurenyu) [inline README.rst]: this is not a full sentence - change "being" to "is"

File: README.rst [hunk 6]
```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
...
+models, with more frameworks coming soon.
+
+Support for using MXNet with Amazon Elastic Inference in SageMaker is supported in the public `SageMaker MXNet containers <https://github.com/aws/sagemaker-mxnet-container>`__.
+
+* For information on how to use the Python SDK to create an endpoint with Amazon Elastic Inference and MXNet in SageMaker, see `Deploying MXNet Models <https://github.com/aws/sagemaker-python-sdk/tree/master/src/sagemaker/mxnet#deploying-mxnet-models>`__.
+* For information on how Amazon Elastic Inference works, see `How EI Works <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-how-it-works>`__.
+* For more information in regards to using Amazon Elastic Inference in SageMaker, see `Amazon SageMaker Elastic Inference <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html>`__.
+* For notebook examples on how to use Amazon Elastic Inference with MXNet through the Python SDK in SageMaker, see `EI Sample Notebooks <https://docs.aws.amazon.com/sagemaker/latest/dg/ei.html#ei-intro-sample-nb>`__.
+
+SageMaker Elastic Inference MXNet container
```
Comment (laurenyu) [inline README.rst]: I would reframe the wording in this section to be about how to build the official SageMaker EI containers

File: README.rst [hunk 7]
```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
...
+
+    # CPU
+    docker build -t preprod-mxnet:1.3.0-cpu-py3 --build-arg py_version=3
+    --build-arg framework_installable=amazonei_mxnet-1.3.0-py2.py3-none-manylinux1_x86_64.whl -f Dockerfile.cpu .
+
+
+* For information about downloading and installing the enhanced binary for Apache MXNet, see `Install Amazon EI Enabled Apache MXNet <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ei-mxnet.html#ei-apache>`__.
+* For information on which versions of MXNet is supported for Elastic Inference within SageMaker, see `MXNet SageMaker Estimators <https://github.com/aws/sagemaker-python-sdk#mxnet-sagemaker-estimators>`__.
+
+Using MXNet with Amazon Elastic Inference
```
Comment (laurenyu) [inline README.rst]: this seems like something that belongs in the Python SDK MXNet README

File: README.rst [hunk 8]
```diff
@@ -273,6 +270,50 @@ To run SageMaker integration tests:
                                       --instance-type ml.m4.xlarge \
                                       --tag 1.3.0-cpu-py3
 
+Amazon Elastic Inference with MXNet in SageMaker
```
Comment (laurenyu) [inline README.rst]: I'd move this to a subsection under "Building Images"

Comment (ChoiByungWook) [reply README.rst]: I don't agree because this is about using MXNet with Elastic Inference in SageMaker and not necessarily how to build the image.

Comment (laurenyu) [reply README.rst]: how to use MXNet with EI (I assume you're referring to the last section) belongs in the Python SDK README, not this repo's.

File: README.rst (hunk 3 above)
Comment (laurenyu) [reply README.rst]: in that case, I'd change it to "available on Amazon S3 at <link>" or "available on Amazon S3 here: <link>"

File: README.rst (hunk 7 above)
Comment (laurenyu) [reply README.rst]: The reference to `model_fn` is still in the context of how to use the container on SageMaker, which isn't relevant to this README. If you think it's important to note here that there is code specific to EI in the MXNet image code, then I would suggest adding a new section that talks about the code in this repository, and referencing it there.

Comment (ChoiByungWook) [reply README.rst]: Fair point. Moving to Python SDK.

File: README.rst (hunk 3 above)
Comment (ChoiByungWook) [reply README.rst]: Done.

File: README.rst (hunk 7 above)
Comment (ChoiByungWook) [reply README.rst]: Created a PR with the doc update for EI in MXNet README. https://github.com/aws/sagemaker-python-sdk/pull/618

File: README.rst (hunk 8 above)
Comment (laurenyu) [reply README.rst]: now that the other section has been moved to the Python SDK repository, I still think this should go under "building". I'd at least move this to be above running tests; it doesn't make sense for this README to talk about building images then testing images then building more images

**Pull request merged**